docs/supported-technologies.asciidoc

```diff
@@ -46,7 +46,7 @@ The Kotlin version refers to the Kotlin gradle plugin versions, also aligned wit
 |11
 
 |Kotlin
-|1.5+
+|1.8+
```

Comment:
I guess this is leftover from the previous PR on upgrading the gradle version?

Reply:
No, it was an unexpected transitive upgrade that came from the OTel Android lib 😅